### ISSUE TITLE
Change date formatting to match UI

### ISSUE THREAD

**Issue by mikeallbutt** [DiscipleTools/disciple-tools-mobile-app#666]
Change date formatting to match UI of the user (start and end dates, baptism etc.)

It is using mm-dd-yy but some people want to use dd-mm-yy

**Comment by squigglybob** [comment]
@zdmc23 or @mikeallbutt is this one closeable?

**Comment by zdmc23** [comment]
We use mmomentjs, and I think we want to go with their implementation and not override it. @mikeallbutt , can we close?

**Comment by mikeallbutt** [comment]
I am pretty sure this needs to be open still since the date format is forcing all users to use the American way of mm-dd-yy and not how all other countries do dates of dd-mm-yy

**Comment by mikeallbutt** [comment]
@zdmc23 this is still an issue and should be done soon IMO 🙂

**Comment by squigglybob** [comment]
@zdmc23 I haven't looked in at the code, but you should be able to make moment use the locale the user is using to format the date according to that locale.

Unfortunately at the moment DT, only has US English, which would have the mm-dd-yy format for dates.

**Comment by zdmc23** [comment]
@squigglybob yes, we do attempt to do that in the `use-i18n` hook (https://github.com/DiscipleTools/disciple-tools-mobile-app/blob/cbe119ae1f68256541d0654f93a9cd668741611f/hooks/use-i18n.js#L166), but not sure why we aren't having the desired outcome. Need to debug it, but just haven't had the chance yet

**Comment by zdmc23** [comment]
Duplicate of #566
